backend/src/modules/lobby/lobbyRoutes.js

```diff
@@ -27,18 +27,33 @@ router.get('/list', async (req, res, next) => {
 router.get('/join-to-lobby-detail/:lobbyHash', async (req, res, next) => {
   const dbConnection = req[DB_CONNECTION_KEY];
   const { lobbyHash } = req.params;
-
   // TODO select lobby detail by lobbyHash and return to FE
-
+  const [lobbyId, invitationId]  = Hashids.decode(lobbyHash);
+  const lobbyDetail = await dbConnection.query(
+    'SELECT lobby.lobby_id, lobby.name, game.name as eventName, count(lobby_user.user_id) as joinedUsers, max_users FROM lobby LEFT JOIN game ON game.game_id = lobby.game_id LEFT JOIN lobby_user ON lobby.lobby_id = lobby_user.lobby_id WHERE lobby.lobby_id = ? AND lobby.active = true;',
+    [lobbyId]
+  );
+  const invitation = await dbConnection.query(
+    'SELECT invitation_id, email FROM invitation WHERE invitation_id = ? AND lobby_id = ?;',
+    [invitationId, lobbyId]
+  );
+  if (!invitation || invitation.length < 1) {
+    return res.status(422).json({
+      error: '422: Invitation expired or token is not valid.',
+    });
+  }
+  const userDetail = await dbConnection.query(
+    'SELECT user_id, email FROM users WHERE email = ? LIMIT 1;',
```

Comment:
Proc tady neni AND active == true?

Reply:
protože jsem původně myslel, že tam nemusí být. Podle mě nutně tam být nemusí, ale dám ho tam, hlavně aby to bylo totožné s tím druhým případem a celý to hodit do jedné funkce.